staging/src/k8s.io/apiserver/pkg/storage/cacher/metrics/metrics.go

```diff
@@ -147,6 +147,17 @@ var (
 		[]string{"resource"},
 	)
 
+	WatchCacheReady = compbasemetrics.NewGaugeVec(
+		&compbasemetrics.GaugeOpts{
+			Namespace:      namespace,
+			Subsystem:      subsystem,
+			Name:           "ready",
```

Comment:
metrics use to have a snake case name more meaninful, we should check with SIG instrumentation for advice

Reply:
@richabanker @dgrisonnet

Reply:
since we concatenate namespace and subsystem to the name of the metric we will end up with:
```
apiserver_watch_cache_ready
```
which I think is an appropriate name here